packages/common/types/howler.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+declare module 'howler';
```

Comment:
can we install howler types instead ? 
https://www.npmjs.com/package/@types/howler